src/utils/system.ts

```diff
@@ -358,11 +358,9 @@ export async function extractZipArchive(
   return new Promise<void>((resolve, reject) => {
     try {
       fs.createReadStream(filePath)
-        .pipe(unzipper.Extract({ strip: 1, path: dir }))
+        .pipe(unzipper.Extract({ path: dir, concurrency: 2 }))
         .on('error', reject)
-        .on('finish', () => {
-          setTimeout(resolve, 100);
-        });
+        .on('close', resolve);
```

Comment:
Do we have test coverage for this change? If not can you paste some console output to prove that it works? 😁

Reply:
Just did a test release to https://github.com/bitsandfoxes/testrepo

This is also on the docs for `unzipper`: https://github.com/ZJONSSON/node-unzipper/#extract-to-a-directory

Reply:
wfm thx